lib/rex/parser/openvas_document.rb

```diff
@@ -24,6 +24,27 @@ def start_element(name=nil,attrs=[])
     case name
     when "host"
       @state[:has_text] = true
+    when 'ref'
+      if in_tag('result') && in_tag('nvt') && in_tag('refs') &&
+         defined?(attrs[0][0]) && defined?(attrs[0][1]) &&
+         defined?(attrs[1][0]) && defined?(attrs[1][1]) &&
+         attrs[1][0] == 'id' && attrs[0][0] == 'type'
+        case attrs[0][1].upcase
+        when 'CVE'
+          @state[:ref_cve] = [] if @state[:ref_cve].nil?
+          @state[:ref_cve].append(attrs[1][1].strip)
+        when 'URL'
+          @state[:ref_url] = [] if @state[:ref_url].nil?
+          @state[:ref_url].append(attrs[1][1].strip)
+        # Uncomment this to import DFN-CERT and CERT-BUND
```

Comment:
This might not be obvious to the end user to uncomment when they want these to be included. Would it be possible to expose an option in the UI to enable/ disable this?

Reply:
Yea I thought that would be a great idea.  I didn't know if I should spend the time developing that part in case you guys didn't want this feature.  Should I make a seperate PR or commit to this one?  Rather new to development with github.  I didn't notice any other arguments for db_import, would something like db_import -a <filename> work?  Or db_import -dfn -cert <filename>?